scripts/figma-sync.js

```diff
@@ -15,6 +15,28 @@ require('dotenv').config();
 const fs = require('fs-extra');
 const path = require('path');
 
+/**
+ * Normalize token names for consistent comparison
+ * - Convert underscores to dashes
+ * - Convert camelCase to dash-separated lowercase
+ * - Handle special cases like 'notificationcard' -> 'notification-card'
+ * @param {string} tokenName - The token name to normalize
+ * @returns {string} - The normalized token name
+ */
+function normalizeTokenName(tokenName) {
+	return tokenName
+		// Convert underscores to dashes
+		.replace(/_/g, '-')
+		// Handle special cases first (before camelCase conversion)
+		.replace(/notificationcard/g, 'notification-card')
+		.replace(/textfield/g, 'text-field')
+		.replace(/inputfield/g, 'input-field')
```

Comment:
I think we need to add page-indicator case too.
```suggestion
		.replace(/inputfield/g, 'input-field')
		.replace(/pageindicator/g, 'page-indicator')
```

Reply:
I've added more cases that we need including `page-indicator`.